clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -826,6 +826,12 @@ DPCT_ENUM_OPTION(
             "prefetch", int(ExperimentalFeatures::Exp_Prefetch),
             "Experimental extension that allows use of SYCL prefetch APIs.\n",
             false),
+        DPCT_OPTION_ENUM_VALUE(
+            "level_zero", int(ExperimentalFeatures::Exp_IPC),
+            "Experimental extension that allows the use of Level Zero APIs "
```

Comment:
Experimental migration feature that allows the use of Level Zero APIs "
            "to migrate the target code like CUDA Inter
            "Process Communication(IPC) APIs".